incubator/blockchain-client-openliberty/README.md

```diff
@@ -1,26 +1,18 @@
 # Blockchain Open Liberty Stack
 
-The Open Liberty stack provides a consistent way of developing microservices based upon the [Jakarta EE](https://jakarta.ee/) and [Eclipse MicroProfile](https://microprofile.io) specifications. This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, that is running on OpenJDK with container-optimizations in OpenJ9.
+The Blockchain Open Liberty stack provides a consistent way of developing microservices based upon the [Jakarta EE](https://jakarta.ee/) and [Eclipse MicroProfile](https://microprofile.io) specifications that consume Blockchain smart contracts deployed on the IBM Blockchain Platform.
```

Comment:
Nit - Obviously we want to promote IBP, but technically we are conformant to any Hyperledger Fabric including the the local dev experience in VSCode.  Do we want to sound open or specific to IBP?